apps/lib/apps.c

```diff
@@ -1982,19 +1976,61 @@ static int do_sign_init(EVP_MD_CTX *ctx, EVP_PKEY *pkey,
         && do_pkey_ctx_init(pkctx, sigopts);
 }
 
-/* Ensure RFC 5280 compliance and then sign the certificate info */
+static int adapt_KID(X509 *cert, const STACK_OF(X509_EXTENSION) *exts,
```

Comment:
Perhaps a better name would be adapt_key_id_ext?

Reply:
I've renamed the aux function to `adapt_keyid_ext`.